src/server/memory/usage.h

```diff
@@ -320,9 +444,56 @@ class ColdObjectTracker
 
   Status OnDelete(ID const& id) { return Self().OnDelete(id); }
 
+ protected:
+  Status SpillPayload_(std::shared_ptr<P>& payload) {
+    assert(payload->is_sealed);
+    util::SpillWriteFile write_file(spill_path);
+    RETURN_ON_ERROR(write_file.Write(payload));
+    RETURN_ON_ERROR(write_file.Sync());
+    BulkAllocator::Free(payload->pointer, payload->data_size);
+    payload->store_fd = -1;
+    payload->pointer = nullptr;
+    payload->is_spilled = true;
+    return Status::OK();
+  }
+
+  Status ReloadPayload_(const ID& id, std::shared_ptr<P>& payload) {
+    assert(payload->is_spilled == true);
+    util::SpillReadFile read_file(spill_path);
+    RETURN_ON_ERROR(read_file.Read(payload, Self().shared_from_this()));
+    return Status::OK();
+  }
+
+  Status DeletePayloadFile_(const ID& id) {
```

Comment:
No need for the `_` suffix, seems that the previous comments hasn't been addressed.